### ISSUE TITLE
polynomial decay learning rate

### ISSUE THREAD

**Issue by eambutu** [chenxi116/TF-deeplab#5]
https://github.com/chenxi116/TF-deeplab/blob/439c872b3d9178d4f28731b25e635c01882193b5/deeplab_model.py#L163

The default end_learning_rate is 0.0001 (https://www.tensorflow.org/api_docs/python/tf/train/polynomial_decay). However, according to https://stackoverflow.com/questions/30033096/what-is-lr-policy-in-caffe/30045244, the default caffe behavior is end_learning_rate=0.
